bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=mcg-osd-deployer
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.18.1
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.18.0+git
```

Comment:
avoid down grading the version

Reply:
This is the version that the controller-gen is pinned to in the repository (which was tested with the current version of operator sdk and was automatically generated by the same): https://github.com/red-hat-storage/mcg-osd-deployer/blob/main/Makefile#L181

Reply:
Done.